bedrock/wordpress/models.py

```diff
@@ -111,6 +121,17 @@ def blog_link(self):
     def blog_name(self):
         return settings.WP_BLOGS[self.wp_blog_slug]['name']
 
+    def get_featured_tag(self, tags):
```

Comment:
@alexgibson this is what I've got so far. `blog_tags` is also added to the context. Usage would be:

```jinja
<span class="featured-tag">{{ article.get_featured_tag(blog_tags) }}</span>
```